src/app/hearing-part/components/hearing-search-table/hearing-search-table.component.scss

```diff
@@ -0,0 +1,14 @@
+#hearing-part-preview {
+  max-height: 30vh;
+  overflow-y: auto;
+}
+
+.clickable {
+  color: #0099FF;
+  text-decoration: underline;
+  cursor: pointer;
+}
+
+.communicationFacilitator {
+  padding-right: 40px;
```

Comment:
I'm not so experienced in developing frontend, but I think we shouldn't use pixels as they don't scale.

Reply:
using pixels is fine in some cases but in this it not seems to. I'm keen to help on this issue but I'd need to know for what its for

Reply:
using pixels is fine in some cases but in this it not seems to. I'm keen to help on this issue but I'd need to know for what its for